setup.py

```diff
@@ -24,5 +24,6 @@
     packages=find_packages(),
     long_description="""\
     An Abstraction library for reading, writing and computing ephemeris data
-    """
+    """,
+    data_files=[('ale', ['ale/config.yml'])]
```

Comment:
Is there a reason why this uses data_files instead of package_data? I'm having a hard time with the setuptools docs.

Reply:
From looking around there doesn't seem to be much of a functional difference between the two other than package_data is used more frequently and is more recommended. No explanation as to why.

Reply:
Can this be moved to package_data then? It also looks less verbose.

Reply:
Done